cpp/include/cudf/column/column_factories.hpp

```diff
@@ -394,6 +394,35 @@ std::unique_ptr<column> make_strings_column(
   cudaStream_t stream                 = 0,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource());
 
+/**
+ * @brief Constructs a LIST type column given offsets column, child column,
+ * and null mask and null count. The columns and mask are moved into the
+ * resulting lists column.
```

Comment:
The `@brief` should be one sentence. Just place the 2nd sentence after a blank line.